vignettes/openxlsx2_style_manual.Rmd

```diff
@@ -82,6 +85,7 @@ wb$add_data(x = mat, colNames = TRUE)
 
 # create a border style and assign it to the workbook
 black <- wb_color(hex = "FF000000")
+# same as wb_color("black") or  wb_color(hex = "#FF000000")
```

Comment:
Probably `wb_color("black")` was not yet around, when I wrote this vignette. Might be a good idea to change this.